### PR TITLE
Bump `typescript` to `4.9.5` and `ts-node` to `10.9.1`

### PR DESCRIPTION
**Pull request by Spacesai1or**
#5675 popped up again, and upgrading `typescript` to `4.9.5` seems to solve the issue

### CODE REVIEW

File: dist/web3.min.js.LICENSE.txt
Comment (Spacesai1or) [inline dist/web3.min.js.LICENSE.txt:1]: I'm not sure why this change was made, but it was a result of running `npm i && npm run build` after bumping `typescript` to `4.9.5`

Comment (luu-alex) [reply dist/web3.min.js.LICENSE.txt:1]: this license file is generated when we changed webpack 4 -> 5

**Comment by luu-alex** [comment]
I believe mosiac tests have been failing for a while now

**Review by luu-alex — APPROVED** [review]

**Comment by Spacesai1or** [comment]
> I believe mosiac tests have been failing for a while now

@luu-alex Yea, seems unrelated to these changes

#6242 to track it

**Review by avkos — APPROVED** [review]
